great_expectations/datasource/fluent/redshift_datasource.py

```diff
@@ -57,32 +55,34 @@ class RedshiftDatasource(SQLDatasource):
         name: The name of this Redshift datasource.
         connection_string: The SQLAlchemy connection string used to connect to the Redshift
             database. This will use a redshift with psycopg2. For example:
-            "redshift+psycopg2://username@host.amazonaws.com:5439/database"
-        connection_details: A RedshiftConnectionDetails object defining the connection details.
+            "redshift+psycopg2://user:password@host.amazonaws.com:5439/database?sslmode=sslmode".
+            Alternatively, a RedshiftConnectionDetails object can be used.
         If connection_details is used, connection_string cannot also be provided.
```

Comment:
You can remove this line since there is no more `connection_details` argument. It looks like we also accept a dictionary whose keys are the connection detail attributes.